Files/ReturnEveryOtherLine/src/main/java/EveryOtherLineReader.java

```diff
@@ -0,0 +1,34 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class EveryOtherLineReader {
+
+    private String filePath;
+
+    public static void main(String[] args) {
+        EveryOtherLineReader everyOtherLineReader = new EveryOtherLineReader("C:/Users/bszcz/OneDrive/data.txt");
```

Comment:
Wszędzie popełniasz ten sam błąd w tych zadaniach, ale wszystkie Ci zaakceptuję, a tylko tutaj odrzuce, abyś miala świadomość teog błędu - jak używasz scieżki absolutnej (czyli takie C:/users[...]) to ja nie mogę tego programu u siebie uruchomić tak łatwo, bo nie mam katalogu "C:/Users/bszcz/" 
![image](https://user-images.githubusercontent.com/17989854/126902865-96284508-f9de-422b-962c-0f34f29f4596.png)


Do poczytania i poprawki:
https://www.google.com/search?q=sciezka+absolutna+i+relatywna&oq=sciezka+absolutna+i+relatywna&aqs=chrome..69i57j0i22i30.4302j0j4&sourceid=chrome&ie=UTF-8

Reply:
rozumiem w czym błąd, ale nie do końca jestem pewna jak go poprawić - czy dobrze mi się wydaje, że zamienienie ścieżki na relatywną załatwi sprawę bo będzie wystarczyło żebyś miał ostatni folder (o tej samej nazwie, np. tutaj "OneDrive") u siebie na komputerze?

Reply:
poprawiłam

Reply:
Głównie chodzi tutaj o ten fragment:
![image](https://user-images.githubusercontent.com/17989854/127038936-9ae58636-3034-42e4-a1b8-deb9f1b90646.png)

https://jaktestowac.pl/lesson/pt1-mk6-s05-05/

>  czy dobrze mi się wydaje, że zamienienie ścieżki na relatywną załatwi sprawę bo będzie wystarczyło żebyś miał ostatni folder (o tej samej nazwie, np. tutaj "OneDrive") 

Nie do końca, bo co z "C:/Users/..." u mnie to wygląda tak:

![image](https://user-images.githubusercontent.com/17989854/127039017-a862a522-2c22-4f2f-82f1-e991cfe2b05d.png)


Czy mam specjalnie robić \Users\bszcz\, aby ten program działał? A co z np. z linuxem gdzie w ogóle nie ma tego przedrostka "C:\"?  Poczytaj ten link wyżej. Jeżeli tam nie znajdziesz odpowiedzi to wygoogluj różnice miedzy ścieżką absolutną i relatywną (względna i bezwzględna) - to nie jest takie cieżkie, bo akurat w tym przypadku nazwa **bardzo dobrze** opisuje różnice :D

Reply:
tak, wydaje mi się że już dobrze poprawiłam, wysłałam do sprawdzenia poniżej